### DISCUSSION TITLE
Windows下PIV只支持RSA2048，NIST P256无法识别

### DISCUSSION THREAD

**Discussion by Editst** [canokeys/canokey-product#3] [Q&A] [answered]
Windows下PIV只支持RSA2048，NIST P256无法识别。
Yubikey可以安装YubiKey-Minidriver解决，但对Canokey无效。

**Comment by dangfan** [comment]
这个需要写 minidriver，但是没找到参考，不知道如何开始……

**Comment by Editst** [comment]
找到了微软提供的一份指南 [Smart Card Minidrivers](https://docs.microsoft.com/en-us/previous-versions/windows/hardware/design/dn631754(v=vs.85)?redirectedfrom=MSDN)

**Reply by dangfan** [reply]
这个没用啊，没写怎么开发

**Comment by Editst** [comment]
https://docs.microsoft.com/zh-cn/windows-hardware/drivers/smartcard/

**Comment by dangfan** [comment, marked as answer]
https://github.com/canokeys/canokey-mini-driver 可以测试这个版本了
